src/main/java/eu/dissco/backend/repository/MongoRepository.java

```diff
@@ -29,19 +29,19 @@ public JsonNode getByVersion(String id, int version, String collectionName)
       throw new NotFoundException(
           "Could not find " + versionId + " in collection: " + collectionName);
     }
-    return mapper.readValue(result.first().toJson(), JsonNode.class).get("eventRecord");
+    return mapper.readValue(result.first().toJson(), JsonNode.class).get("prov:Entity").get("prov:value");
```

Comment:
you repeat "prov:Entity", "prov:value", and "ods:version" a few times in this class, maybe make then constants

Reply:
:100: